src/components/ProfileSettings.js

```diff
@@ -76,6 +77,7 @@ class ProfileSettings extends Component {
           console.log(this.state);
```

Comment:
Delete consoles

Reply:
@AJAkimana Thanks for the comment. I removed the log